tests/validation_tests/test_failures.py

```diff
@@ -47,3 +48,100 @@ def test_serialise_user_centered_failures():
+    assert len(output["Programme Progress"]["Projects Progress Summary"]) == 3
+    assert "PSI" in output
+    assert "Private Sector Investment" in output["PSI"]
+    assert len(output["PSI"]["Private Sector Investment"]) == 2
+    assert "Funding Profiles" in output
+    assert "Project Funding Profiles" in output["Funding Profiles"]
+    assert len(output["Funding Profiles"]["Project Funding Profiles"]) == 1
+    assert "Project Outputs" in output
+    assert "Project Outputs" in output["Project Outputs"]
+    assert len(output["Project Outputs"]["Project Outputs"]) == 1
+    assert "Outcomes" in output
+    assert "Outcome Indicators (excluding footfall) and Footfall Indicator" in output["Outcomes"]
+    assert len(output["Outcomes"]["Outcome Indicators (excluding footfall) and Footfall Indicator"]) == 1
+    assert output == {
+        "Programme Progress": {
+            "Projects Progress Summary": [
+                "For column Start Date you entered text when we expected a date. "
+                "You must enter the start date in the correct format, for example, Dec-22, Jun-23",
+                "For column Completion Date you entered text when we expected a date. "
+                "You must enter the start date in the correct format, for example, Dec-22, Jun-23",
+                "For column Date of Most Important Upcoming Comms Milestone you entered text when we expected a date. "
+                "You must enter the start date in the correct format, for example, Dec-22, Jun-23",
+            ]
+        },
+        "PSI": {
+            "Private Sector Investment": [
+                "For column Private Sector Funding Required you entered text when we expected a number. "
+                "You must enter the required data in the correct format, for example, £5,588.13 or £238,062.50",
+                "For column Private Sector Funding Secured you entered text when we expected a number. "
+                "You must enter the required data in the correct format, for example, £5,588.13 or £238,062.50",
+            ]
+        },
+        "Funding Profiles": {
+            "Project Funding Profiles": [
+                "Between columns G-Y you entered text when we expected a number. "
+                "You must enter the required data in the correct format, for example, £5,588.13 or £238,062.50"
+            ]
+        },
+        "Project Outputs": {
+            "Project Outputs": [
+                "Between columns E-W you entered text when we expected a number. "
+                "You must enter data using the correct format, for example, 9 rather than 9m2. Only use numbers"
+            ]
+        },
+        "Outcomes": {
+            "Outcome Indicators (excluding footfall) and Footfall Indicator": [
+                "Between columns F-O you entered text when we expected a number. "
+                "You must enter data using the correct format, for example, 9 rather than 9m2. Only use numbers"
+            ]
+        },
+    }
```

Comment:
I think it will be best to just have a single assertion on the full output. The initial granularity is a lot of repetition that I don't think makes it easier to debug.